application/src/main/java/org/togetherjava/tjbot/commands/tags/TagsCommand.java

```diff
@@ -49,20 +49,27 @@ public TagsCommand(TagSystem tagSystem) {
 
     @Override
     public void onSlashCommand(@NotNull SlashCommandEvent event) {
-        Collection<String> tagIds = tagSystem.getAllIds();
+
+        // TODO A list might be better than comma separated, which is hard to read
+
+      Collection<String> tagIds = tagSystem.getAllIds();
         if (tagIds.size() > MAX_TAGS_THRESHOLD_WARNING) {
             // TODO Implement the edge case
 
             logger.warn(
                     "The amount of tags is very high and it might soon exceed the maximum character limit. The code should be adjusted to support this edge case soon.");
         }
-        String tagListText =
+       String tagListText =
```

Comment:
There seems to be a minor alignment issue with one space too much or too less. Run Spotless.